common/app/views/fragments/inlineSvg.scala.html

```diff
@@ -1,4 +1,4 @@
-@(file: String, path: String = "", classes: Seq[String] = List())
-<span class="inline-@file @if(path){inline-@path} @classes.mkString(" ")">
+@(file: String, path: String = "", classes: Seq[String] = List(), attributes: Map[String, String] = Map())
```

Comment:
Originally I was going to do that but you never know when we might want to use a `data-` attribute or `aria`. So I went with this instead as it's a little more future proof.